examples/platforms/da15000/radio.c

```diff
@@ -95,15 +95,17 @@ void da15000RadioInit(void)
 void otPlatRadioGetIeeeEui64(otInstance *aInstance, uint8_t *aIeeeEui64)
 {
     (void)aInstance;
+    uint32_t *timeStamp  = (uint32_t *) 0x7F8EA08;
+    uint32_t *testId     = timeStamp + 1;
 
     aIeeeEui64[0] = 0x80;    //80-EA-CA is for Dialog Semiconductor
     aIeeeEui64[1] = 0xEA;
     aIeeeEui64[2] = 0xCA;
-    aIeeeEui64[3] = 0x00;
-    aIeeeEui64[4] = (HARDCODED_NODE_ID >> 24) & 0xff;
-    aIeeeEui64[5] = (HARDCODED_NODE_ID >> 16) & 0xff;
-    aIeeeEui64[6] = (HARDCODED_NODE_ID >>  8) & 0xff;
-    aIeeeEui64[7] =  HARDCODED_NODE_ID        & 0xff;
+    aIeeeEui64[3] = (*testId    >>  8) & 0xff;
+    aIeeeEui64[4] = (*timeStamp >> 24) & 0xff;
+    aIeeeEui64[5] = (*timeStamp >> 16) & 0xff;
+    aIeeeEui64[6] = (*timeStamp >>  8) & 0xff;
+    aIeeeEui64[7] =  *timeStamp        & 0xff;
```

Comment:
As clarified before for volume production it will be responsibility of Dialog's customers to generate unique EUI64. 

In order to simplify current certification verification process we would like to provide an approach, which is generating unique ID during a whole life cycle of DA15000 after it leaves the factory. It is verified that all data taken from timestamp are enough to make it unique amongst the population of Engineering Sample devices . Selected data are chosen by purpose.